lib/games/checkDeveloperProductName.js

```diff
@@ -23,17 +23,25 @@ exports.func = (args) => {
   const productId = parseInt(args.productId) ? parseInt(args.productId) : 0
 
   return http({
-    url: '//www.roblox.com/places/check-developerproduct-name?universeId=' + universeId + '&developerProductId=' + productId + '&developerProductName=' + args.productName + '&_=1515792139751',
+    url: `//apis.roblox.com/developer-products/v1/universes/${universeId}/developerproducts?pageNumber=1&pageSize=2147483647`,
     options: {
       method: 'GET',
-      jar,
+      jar: jar,
       resolveWithFullResponse: true
     }
   }).then((res) => {
     if (res.statusCode !== 200) {
       throw new Error('You are not logged in')
     } else {
-      return JSON.parse(res.body)
+      const data = JSON.parse(res.body)
+      let exists = false
+      data.forEach(product => {
+        if(product.name == args.productName || product.id == productId){
+          exists = true
+          return
```

Comment:
This returns undefined and the typing says it returns a boolean.
Would a better approach be:
```js
for (const product of data) {
   if (...) {
       return true

}
}
return false
```?